### PR TITLE
More frontend stuff

### PR DESCRIPTION
**Pull request by T0RAT0RA**

### CODE REVIEW

File: AlayaCare Cloud Tech Radar.csv [hunk 1]
```diff
@@ -25,3 +26,4 @@
 "Vue 3","Trial","languages-and-frameworks","FALSE","<p>Vue 3 has been release in September 2020. Migrating vue applications to this release is highly dependent of its plugins support of Vue 3.</p>"
 "Vue CLI","Adopt","languages-and-frameworks","FALSE","<p><a href='https://cli.vuejs.org/'>Vue CLI</a> offers standad tooling for Vue.js development.</p>"
```
Comment (et1421) [inline AlayaCare Cloud Tech Radar.csv]:
```suggestion
"Vue CLI","Adopt","languages-and-frameworks","FALSE","<p><a href='https://cli.vuejs.org/'>Vue CLI</a> offers standard tooling for Vue.js development.</p>"
"Vue Composition API","languages-and-frameworks","FALSE","<p><a href='https://v3.vuejs.org/guide/composition-api-introduction.html'>Vue Composition API</a>allows us to extract repeatable parts of the interface coupled with its functionality into reusable pieces of code.</p>"
```

File: AlayaCare Cloud Tech Radar.csv [hunk 2]
```diff
@@ -1,15 +1,16 @@
 "name","ring","quadrant","isNew","description"
+"Cypress","Adopt","languages-and-frameworks","FALSE","<p></p>"
+"Micro frontend","Trial","Techniques","FALSE","<p></p>"
```
Comment (et1421) [inline AlayaCare Cloud Tech Radar.csv]: Weren't we following alphabetical order before ?

Comment (T0RAT0RA) [reply AlayaCare Cloud Tech Radar.csv]: This is to fix the ring order. their tool will define the ring in order of appearance in the file, so `Adopt` needs to appear first in the data.

Comment (et1421) [reply AlayaCare Cloud Tech Radar.csv]: Shouldn't we regroup all categories together then ?

**Pull request merged**